fog/ingest/server/src/controller.rs

```diff
@@ -1336,3 +1403,19 @@ where
         Ok(())
     }
 }
+
+// Helper function which extracts ingress pubkey bytes from (our own) report, to
+// double check
+fn try_extract_ingress_pubkey_from_report(
```

Comment:
this code is mostly the same as what we added to fog-report-cli last week-ish

Reply:
we should probably share this code in those two places, but i'm actually not sure what would be a good place to put it, and i think the fog-report-cli change doesn't exist in the release branch

Reply:
I agree this should not be duplicated. Maybe in the [API](https://github.com/mobilecoinfoundation/fog/tree/master/fog/api/src) crate?

Reply:
sounds good, i will put it there

Reply:
i ran `cargo tree` in `fog-api` crate, it already depends on `mc-attest-ake` and `mc-attest-core` anyways